frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -141,7 +136,25 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 				#frame_support::sp_std::marker::PhantomData<(#type_use_gen,)>,
 				#frame_support::Never,
 			),
-			#( #( #[doc = #fn_doc] )* #fn_name( #( #args_compact_attr #args_type ),* ), )*
+			#(
+				#( #[doc = #fn_doc] )*
+				#fn_name {
+					#( #args_compact_attr #args_name: #args_type ),*
+				},
+			)*
+		}
+
+		impl<#type_impl_gen> #call_ident<#type_use_gen> #where_clause {
+			#(
+				#[doc = #new_call_variant_doc]
+				pub fn #new_call_variant_fn_name(
```

Comment:
For minimizing changes in benchmarking macros because of changing call variants from tuples to structs. That would have required constructing call variants with field names.

Reply:
Ahh okay, hadn't see it there. I thought this wasn't used.